front/src/common/BootstrapSNCF/InputGroupSNCF.tsx

```diff
@@ -29,6 +29,7 @@ type Props = {
   min?: number;
   max?: number;
   step?: number | string;
+  textAlignment?: 'left' | 'right';
```

Comment:
if there is no centered value, i would have named it textRight, because left is the standard behaviour, and use it as a boolean

Reply:
ok